proto/Cargo.toml

```diff
@@ -31,7 +31,7 @@ all-features = true
 #
 # Sometimes cleaning the build cache with `cargo clean` might be necessary to solve
 # issues related to outdated generated files.
-default = ["grpc"]
+default = ["grpc", "serde"]
```

Comment:
Are you sure we want to have it by default? I feel like we don't.

Reply:
Disabling this by default would break backwards-compatibility.

Reply:
So we supported serde in previous versions by default without feature? Platform could use feature if we need this in case we dump requests

Reply:
removed serde from default features